### PR TITLE
Uploaded camera communications sign off

### PR DESCRIPTION
**Pull request by TycheThePanda**

### CODE REVIEW

**Comment by kesternucum** [comment]
Eve was having a bit of trouble with getting GitHub logged into her Linux machine (since it requires a key) so I pushed up the branch on her behalf, but she is the author of this signoff.

File: Documentation/Signoffs/camcommdetaildesign.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Camera Communications Detail Design
```
Comment (JesseTNRoberts) [inline Documentation/Signoffs/camcommdetaildesign.md:1]: The files for the images should not be in the wild in the signoff folder. House them somewhere to keep things organized.

File: Documentation/Signoffs/camcommdetaildesign.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+
+1. The camera communications subsystem will interact directly with each camera as part of its build. This subsystem will allow each individual camera to connect to the small LAN(Local Area Network) that is provided by the server and its corresponding communications subsystem. This implies a need to provide each camera with the capability of connecting to WiFi. The chosen cameras have only ethernet (RJ45) inputs and thus require an adapter to connect them to a WiFi network.
+
+
+## Specifications
+
+![](vonetsadapter.jpg "Vonets Network Bridge")
+
+1. Chosen Device
```
Comment (JesseTNRoberts) [inline Documentation/Signoffs/camcommdetaildesign.md:15]: How can you choose the device? You haven't said what it is to do yet.

File: Documentation/Signoffs/camcommdetaildesign.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+
+## Big Picture
+
+
+
+1. The camera communications subsystem will interact directly with each camera as part of its build. This subsystem will allow each individual camera to connect to the small LAN(Local Area Network) that is provided by the server and its corresponding communications subsystem. This implies a need to provide each camera with the capability of connecting to WiFi. The chosen cameras have only ethernet (RJ45) inputs and thus require an adapter to connect them to a WiFi network.
+
+
+## Specifications
```
Comment (JesseTNRoberts) [inline Documentation/Signoffs/camcommdetaildesign.md:11]: No consideration of distance. Why are the capabilities of this module being given where the system constraints are meant to go?

**Comment by TycheThePanda** [comment]
Closing this branch in favor of new camera communications branch.

**Pull request closed without merging**